UnityProject/Assets/Scripts/InGameEvents/InGameEventScripts/EventAsteroids.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace InGameEvents
+{
+	public class EventAsteroids : EventScriptBase
+	{
+		private MatrixInfo StationMatrix;
```

Comment:
Please, check the naming, so that all private variables and non-public fields except for constants / constant-like ones would be in camelCase and properties, public fields, classes, structs, enum constants, and methods would be in PascalCase
https://unitystation.github.io/unitystation/contribution-guides/Development-Standards-Guide/#code-style

Reply:
think ive fixed them all now.